src/wpt/harness.ts

```diff
@@ -230,6 +280,7 @@ globalThis.assert_throws_dom = (
     func = descriptionOrFunc;
     description = maybeDescription;
   } else {
+    // @ts-ignore
```

Comment:
It seems eslint is not running on CI for this file. If it did, it would give an error about replacing ts-ignore with ts-expect-error